### ISSUE TITLE
[BUG] no-contradicting-classname throw unexpected error

### ISSUE THREAD

**Issue by johnxussw** [francoismassart/eslint-plugin-tailwindcss#195]
**Describe the bug**
```bash
 52:11  error  Classnames bg-[url('/example.jpg')], bg-center are conflicting!  tailwindcss/no-contradicting-classname
```

**To Reproduce**
1. add bg-center bg-[url(/example.jpg)] on same HTML tag
2. npx eslint . --ext .ts,.tsx

**Expected behavior**
No warning or error when we use *bg-center* and **bg-[url(*.jpg)]* together

**Screenshots**

**Environment (please complete the following information):**
 - OS: windows 11
 - Softwares + version used:
     -  Node 16.17.0
     -  eslint-plugin-tailwindcss 3.8.0 
     -  tailwindcss 3.1.8

**Additional context**

**eslint config file or live demo**
By providing a link to a live demo, a demo video or a github repo fixing the issue will be easier.

**Comment by TomasSestak** [comment]
@johnxu77 why did you close the issue?

**Comment by johnxu77** [comment]
A duplicated issue #186 exists
